src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -557,7 +557,7 @@ public boolean updateMemberCache(MemberImpl member, boolean forceRemove)
             GuildVoiceStateImpl voiceState = (GuildVoiceStateImpl) member.getVoiceState();
             if (voiceState != null)
             {
-                VoiceChannelImpl connectedChannel = (VoiceChannelImpl) voiceState.getChannel();
+                AudioChannelMixin<?> connectedChannel = (AudioChannelMixin<?>) voiceState.getChannel();
                 if (connectedChannel != null)
                     connectedChannel.getConnectedMembersMap().remove(member.getIdLong());
```

Comment:
```suggestion
                AudioChannel connectedChannel = voiceState.getChannel();
                if (connectedChannel instanceof AudioChannelMixin)
                    ((AudioChannelMixin<?>) connectedChannel).getConnectedMembersMap().remove(member.getIdLong());
```

Reply:
Cannot resolve method 'getConnectedMembersMap' in 'AudioChannel'